### DISCUSSION TITLE
Track file downloads using the download attribute

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#2707] [Anything Else]
Plausible could look for the file extension in not only the href attribute but also the download attribute. We're creating file download links something like this:

`<a href="blob:https://temporaryurl/" download="file.jpg">`

(via email)

**Comment by louis77** [comment]
Same issue here.  All our file links have URLs that do not show the filename.

Perhaps we could also make use of a script parameter like this

```html
<script defer file-types-regex="\/files\/\?id=[0-9]*" 
              data-domain="yourdomain.com" 
              src="https://plausible.io/js/script.file-downloads.js"></script>
```
